hugo/content/Configuration/tls.md

```diff
@@ -117,7 +117,8 @@ setip()
 By default, the pgo client will trust certificates issued by one of the
 Certificate Authorities listed in the operating system's default CA trust
 store. To prevent their inclusion, two configuation methods are available:
-either the Bash environment variable
+either the Bash environment variable. Windows clients have no default CA
+trust and thus have this setting configured to `true` by default.
```

Comment:
This is an awkward placement for this sentence. `either the Bash environment variable` is the beginning of a list that continues `or the --exclude-os-trust can be included during client use`.

My inclination is to get vague, drop this sentence, and add `if any` to the first sentence in this paragraph:

> By default, the pgo client will trust certificates issued by one of the Certificate Authorities listed in the operating system's default CA trust store, if any.

Reply:
1) Yes, that placement was terrible.
2) My reservation on being too vague was in response to needing to explicitly disable this in the 4.2.0 release for windows clients and no longer needing it going forward. That said, I agree with your suggestions and have updated.